graphql/util.go

```diff
@@ -772,21 +771,21 @@ func groupProjects(projects []model.ProjectRef, onlyDefaultedToRepo bool) ([]*Gr
 	return groupsArr, nil
 }
 
-func hasProjectPermission(ctx context.Context, resource string, next graphql.Resolver, permissionLevel int) (res interface{}, err error) {
-	user := gimlet.GetUser(ctx)
-	if user == nil {
-		return nil, Forbidden.Send(ctx, "user not logged in")
+func getProjectIdFromArgs(ctx context.Context, args map[string]interface{}) (res string, err error) {
```

Comment:
Can we add some comments to this since it might not be clear that this applies to directive args.